react/Product.js

```diff
@@ -73,15 +73,22 @@ const formatGTIN = (gtin) => {
 const parseSKUToOffer = (
   item,
   currency,
-  { decimals, pricesWithTax, useSellerDefault }
+  { decimals, pricesWithTax, useSellerDefault, gtinValue }
 ) => {
   const seller = useSellerDefault
     ? getSellerDefault(item.sellers)
     : lowHighForSellers(item.sellers, { pricesWithTax }).low
 
+  const { low: lowSeller } = lowHighForSellers(item.sellers, { pricesWithTax })
```

Comment:
Avoid duplicate calls to `lowHighForSellers` when `useSellerDefault` is false:

```suggestion
  const lowSeller = lowHighForSellers(item.sellers, { pricesWithTax }).low
  const seller = useSellerDefault ? getSellerDefault(item.sellers) : lowSeller
```